mlf_core/sync/sync.py

```diff
@@ -97,8 +102,9 @@ def inspect_sync_dir(self):
         if not os.path.exists(os.path.join(str(self.project_dir), '.mlf_core.yml')):
             print(f'[bold red]{self.project_dir} does not appear to contain a .mlf_core.yml file. Did you delete it?')
             sys.exit(1)
-        # store .mlf_core.yml content for later reuse in the dry create run
+            # store .mlf_core.yml content for later reuse in the dry create run
```

Comment:
Why?

Reply:
Seems like auto formatting is trolling again